src/emulator/auth/operations.ts

```diff
@@ -1597,6 +1594,136 @@ function listVerificationCodesInProject(
   };
 }
 
+function mfaEnrollmentStart(
+  state: ProjectState,
+  reqBody: Schemas["GoogleCloudIdentitytoolkitV2StartMfaEnrollmentRequest"]
+): Schemas["GoogleCloudIdentitytoolkitV2StartMfaEnrollmentResponse"] {
+  assert(reqBody.idToken, "MISSING_ID_TOKEN");
+  assert(reqBody.phoneEnrollmentInfo, "((Missing phoneEnrollmentInfo.))");
+  // recaptchaToken, safetyNetToken, iosReceipt, and iosSecret are intentionally
+  // ignored because the emulator doesn't implement anti-abuse features.
+  // autoRetrievalInfo is ignored because SMS will not actually be sent.
+
+  const phoneNumber = reqBody.phoneEnrollmentInfo.phoneNumber;
+  assert(phoneNumber, "((Missing phoneNumber.))");
+  assert(isValidPhoneNumber(phoneNumber), "((Invalid phone number.)");
```

Comment:
Nit: missing closing `)` in assertion message
```suggestion
  assert(isValidPhoneNumber(phoneNumber), "((Invalid phone number.))");
```